tests/virtual_machines_test.go

```diff
@@ -43,7 +45,7 @@ var _ = Describe("Virtual Machines", func() {
 				Expect(err).ToNot(HaveOccurred())
 				return len(vmList.Items)
 
-			}, 30*time.Second, 3*time.Second).Should(Equal(0), "failed to remove all vm objects")
+			}, 60*time.Second, 3*time.Second).Should(Equal(0), "failed to remove all vm objects")
```

Comment:
lets configure this times as const in the file so it will be easier to change them in the future